test/fitnesse/wikitext/parser/ContentsItemTest.java

```diff
@@ -47,6 +47,12 @@ public void buildsRegraced() throws Exception {
         assertBuildsOption("PlainItem", new String[]{}, "-g", "REGRACE_TOC", "<a href=\"PlainItem\" class=\"static\">Plain Item</a>");
     }
 
+  @Test
+  public void buildsTestPageCount() throws Exception {
+    assertBuildsOption("PlainItem", new String[]{"Suite=true"}, "-c", "TEST_PAGE_COUNT_TOC",
+            "<a href=\"PlainItem\" class=\"suite\">PlainItem(Test pages:0)</a>");
```

Comment:
```suggestion
            "<a href=\"PlainItem\" class=\"suite\">PlainItem (Test pages: 0)</a>");
```

Please also add tests for a test page and a filled suite (including non-test pages like SetUp, ScenarioLibrary, and TearDown which should not be counted).

Thinking of what should be counted: how are SuiteSetUp and SuiteTearDown pages handled? Can you ensure that they are counted the same as in an actual test run (i.e. count them if and only if they are counted as tests when a suite is run)

Reply:
Thanks. SuiteSetUp and SuiteTearDown are counted as test when suite is run. So, I added logic to show that as count. Also added the requested tests.